### PR TITLE
feat: add logic of connection metamask

### PR DESCRIPTION
**Pull request by zhangjing28**
## Description

* add logic of connection metamask

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [ ] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [X] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [ ] Update the version number in `package.json` and `src/config/common.json`
- [ ] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [ ] Update `CHANGELOG.md`
- [ ] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

File: src/pages/SBTPage/components/MintPanel/index.tsx [hunk 1]
```diff
@@ -236,64 +210,43 @@ const MintPanel = () => {
...
         <WatermarkSwiper />
         <div className="flex flex-col flex-1">
-          <div className="bg-secondary rounded-lg  ml-6 pb-4">
-            <div className="text-white text-opacity-60 border-b border-split p-4 flex ">
-              {/* 
+          {!ethAddress && (
+            <div className="bg-secondary rounded-lg  ml-6 pb-4 font-red-hat-mono font-medium text-sm">
+              <div className="text-white text-opacity-60 border-b border-split p-4 flex ">
+                {/* 
               // @ts-ignore */}
```
Comment (jincan39) [inline src/pages/SBTPage/components/MintPanel/index.tsx:217]: unnecessary code

File: src/pages/SBTPage/components/MintPanel/index.tsx [hunk 2]
```diff
@@ -236,64 +210,43 @@ const MintPanel = () => {
...
-                </svg>
-              )}
-              Apply all
+          )}
+          {!ethAddress && (
+            <div className="bg-secondary rounded-lg mt-4 ml-6 pb-4">
+              <div className="text-white text-opacity-60 border-b border-split p-4 flex font-red-hat-mono font-medium text-sm">
+                Connect your MetaMask to unlock more Crypto Watermarks
+              </div>
+              <MetamaskButton />
```
Comment (jincan39) [inline src/pages/SBTPage/components/MintPanel/index.tsx:244]: image
this block should move to above block.

**Review by jincan39 — APPROVED** [review]
LGTM

File: src/pages/SBTPage/components/MintPanel/index.tsx (hunk 2 above)
Comment (zhangjing28) [reply src/pages/SBTPage/components/MintPanel/index.tsx:244]: Will do in the next PR

File: src/pages/SBTPage/components/MintPanel/index.tsx (hunk 1 above)
Comment (zhangjing28) [reply src/pages/SBTPage/components/MintPanel/index.tsx:217]: It will cause an error if removed

**Pull request merged**